VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -177,6 +186,10 @@ def device_list_for_billing(self):
             Since root devices are at mount points '/' and '/boot/efi' we use file_system_info to find the root_devices based on the mount points.
         '''
 
+        # check if user off the sudo usage in commands
+        self.sudo_off = self.hutil.get_intvalue_from_configfile("sudo_off", self.sudo_off)
```

Comment:
does this use default value (0?) if conf does not have the setting?

Reply:
Yes